chaos-monkey-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -23,7 +23,11 @@ There are two ways to integrate the Chaos Monkey for Spring Boot into your exist
 [source,txt,subs="verbatim,attributes"]
 .command-line
 ----
+# spring profile
 java -jar your-app.jar --spring.profiles.active=chaos-monkey
+
+# system property
+java -DCHAOS_MONKEY=enabled -jar your-app.jar
```

Comment:
```suggestion
java -jar your-app.jar -DCHAOS_MONKEY=enabled
```

Reply:
I think the property name LOAD_CHAOS_MONKEY=true would suit better

Reply:
Hi @WtfJoke 
thank you for reviewing my request.
i think the naming you suggested helps us improve code readability.
so I immediately fixed it in the code.

Reply:
Thanks!